arangod/Aql/FixedVarExpressionContext.cpp

```diff
@@ -62,7 +62,8 @@ void FixedVarExpressionContext::serializeAllVariables(velocypack::Options const&
   for (auto const& it : _vars) {
     builder.openArray();
     it.first->toVelocyPack(builder);
-    it.second.toVelocyPack(&opts, builder, true);
+    it.second.toVelocyPack(&opts, builder, /*resolveExternals*/false,
```

Comment:
I agree with @jsteemann is this intentional and does it not modify the behavior?
A bug here will be very hard to find as a lot needs to fall in place to actually get an external into this position.